lib/modules/datasource/conda/index.spec.ts

```diff
@@ -61,5 +62,41 @@ describe('modules/datasource/conda/index', () => {
       expect(res).toMatchSnapshot();
       expect(res?.releases).toHaveLength(94);
     });
+
+    it('returns null without registryUrl', async () => {
+      const condaDatasource = new CondaDatasource();
+      const res = await condaDatasource.getReleases({
+        registryUrl: '',
+        packageName: depName,
+      });
+      expect(res).toBeNull();
+    });
+
+    it('supports multiple custom datasource urls', async () => {
+      const depName = 'pytest';
+      httpMock
+        .scope('https://api.anaconda.org/package/rapids')
+        .get(`/${depName}`)
+        .reply(404);
+      //.replyWithError('error');
+      httpMock
+        .scope('https://api.anaconda.org/package/conda-forge')
+        .get(`/${depName}`)
+        .reply(200, Fixtures.get('pytest.json'));
```

Comment:
can we use a much smaller fixture?

Reply:
@viceice  I'm not sure, what do you mean by smaller fixture ?
What's the issue using the same fixture from an other test ?

Reply:
beest would be to copy the file inline and remove most versions. so leave only a couple of them ( eg 5). makes validation also much easier.

the main concern is file IO, which is expensive on windows (we've now more than 1000 tests), so we should try to avoid it on tests.

Reply:
@viceice  ok ! thanks for this explanation.
It's now fixed